ev3dev2/motor.py

```diff
@@ -1857,6 +1866,9 @@ def __init__(self, left_motor_port, right_motor_port, desc=None, motor_class=Lar
         # color sensor used by follow_line()
         self.cs = None
 
+        # gyro sensor used by follow_gyro_angle()
+        self.gyro = None
```

Comment:
How does the gyro get initialized?

Reply:
I see from the examples it seems the intent is for the user to set it themselves. While this might be fine in general, I worry it will cause problems because a) we don't do this elsewhere AFAIK and b) it probably won't show up well in the documentation. I'm thinking it might be better to have a setter method or a property for it.

Reply:
You're correct, the intent for the user to set it. I followed the same pattern used for the color sensor `cs` attribute. I can make `gyro` an explicit property, but will that make it inconsistent though?

Reply:
Hmmm... That seems odd to me. Relatedly, I'm a bit bugged by the fact that we're putting so many different optional things in the `MoveTank` class. This isn't an issue specific to your PR; I'm just thinking through how we really want to do it.

How about, at least for now, we make it a property (getter/setter pair with `_gyro` private member internally) with a docstring so that it shows up in the documentation, and also change the color sensor to match? I don't think it will break anyone's existing code.

Reply:
That's a good idea. I'll work on making them both properties in a non-breaking way.